Earthfile

```diff
@@ -456,7 +456,7 @@ uki-tools-image:
 # This is for easy testing SecureBoot locally for development purposes
 # Installing this keys in other place than a VM for testing SecureBoot is irresponsible
 uki:
-    FROM ubuntu
+    FROM +uki-tools-image
```

Comment:
I think this is the only mention of `uki-tools-image` target, should we get rid of the target?